### ISSUE TITLE
Chrome logo is shown in Dark mode in the brave://settings page in 1.55.17 

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#31355]
## Description 

Issue is reported by @rebron 

Chrome logo is shown in Dark mode in the brave://settings page in 1.55.17 

## Steps to Reproduce

**Case 1:**

   1. Clean profile 1.55.17
   2. Open brave://settings/appearance
   3. Select `Dark` mode
   4. Click on the `Extension` via left settings menu
   5. Click on the `Manage extensions`

**Case 2:**

   1. Clean profile 1.55.17
   2. Set the browser view in narrow mode
   3. Open brave://settings/appearance
   4. Select `Dark` mode
   5. Click on the `Extension` via left settings menu
   6. Click on the `Manage extensions`

## Actual result:

Case 1:
![image](https://github.com/brave/brave-browser/assets/38657976/d93558fe-842e-4f1a-82ff-0ddd61eb5cbc)

Case 2:

![image](https://github.com/brave/brave-browser/assets/38657976/dda1903f-c42c-4a44-aa60-98d7f9b0a46e)

## Expected result:
Chrome logo should not be shown 

## Reproduces how often: 

Easy
## Brave version (brave://version info)

Brave | 1.55.17 Chromium: 115.0.5790.40 (Official Build) nightly (64-bit)
-- | --
Revision | 071c9ddea889c3c7887daf4eac13fed72d4fff62-refs/branch-heads/5790@{#979}
OS | Windows 10 Version 22H2 (Build 19045.3086)

## Version/Channel Information:

- Can you reproduce this issue with the current release? No 
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @rebron @rebron @brave/qa-team

**Comment by rebron** [comment]
<img width="298" alt="Screenshot 2023-06-27 at 4 40 39 PM" src="https://github.com/brave/brave-browser/assets/5951041/7e0dae62-6846-441f-8832-6bc82c0702e1">
In 1.55.17 macOS going to hamburger menu in brave://settings

**Comment by GeetaSarvadnya** [comment]
Added case 2 for narrow view.

**Comment by GeetaSarvadnya** [comment]
- The issue **NOT** reproducible in `1.56.1` 
- Issue is Reproducible only in `1.57.1` and  `1.58.2` 


 1.57.1 | 1.58.2
-------|--------
![image](https://github.com/brave/brave-browser/assets/38657976/62775a31-8f33-4601-88c3-a495b187aaa7) | ![image](https://github.com/brave/brave-browser/assets/38657976/ccefedce-b755-4bfd-97bc-aff409f81370)

**Comment by bsclifton** [comment]
AFAIK, this only happens if you click the burger icon. The issue is that should NOT be displaying in the header
![burger](https://github.com/brave/brave-browser/assets/4733304/9b88f9b1-8d8d-4d1b-923a-ac012b494e24)

**Comment by MadhaviSeelam** [comment]
Additionally, chrome logo is shown in Dark mode only for `brave://management` both in 1.58.52 & 1.57.27 & 1.56.14 in Win11 x64


1.56.14 | 1.58.52
----- | ----
<img width="608" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/2336e3d7-1a80-412c-86fe-0c802702409c">|<img width="621" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/78ff0652-de9c-4da2-b585-9cdb67b435d9">

**Comment by LaurenWags** [comment]
fwiw the logo on brave://management is logged with https://github.com/brave/brave-browser/issues/25807

**Comment by fallaciousreasoning** [comment]
Just had a quick look - they both have the same root cause which is a special picture source set in dark mode pointing to a different SVG. So the good news is fixing one should fix both.

Changing the image served by `//resources/images/chrome_logo_dark.svg` should fix this :smile:

I'm going to add a `good first issue` label, because someone was asking me about some more.

**Comment by ItIsOHM** [comment]
Thanks a lot @fallaciousreasoning! I'll start working on this ASAP :D

**Comment by ItIsOHM** [comment]
Hey there @fallaciousreasoning, I'm sorry but I'm not able to find the path that you mentioned either in `brave-browser` or in `brave-core`. I have cloned both the repos but my guess is I need to build brave following the instructions [here?](https://github.com/brave/brave-browser/blob/master/README.md#build-brave). The guide also involves building Brave by first building Chromium, which I'm unsure as to why that part is necessary.

Sorry if these are dumb questions but I'm only a beginner here :D

**Comment by fallaciousreasoning** [comment]
Sorry, missed these comments @ItIsOHM.

If you follow the [instructions for cloning](https://github.com/brave/brave-browser/blob/master/README.md#clone-and-initialize-the-repo) 

```
git clone git@github.com:brave/brave-core.git path-to-your-project-folder/src/brave
cd path-to-your-project-folder/src/brave
npm install

# the Chromium source is downloaded, which has a large history (gigabytes of data)
# this might take really long to finish depending on internet speed

npm run init
```

that will download the Chromium source code which Brave uses as part of it's build process.

`//resources/images/chrome_logo_dark.svg` isn't a file path sorry, it's a resource served from `chrome://resources/images/chrome_logo_dark.svg`. I think the actual file will be located at `/path/to/your/project/folder/src/ui/webui/resources/images/chrome_logo_dark.svg` (but only after you've run `npm run init`)

When you clone Brave, it's important that you have a `src` folder up one level from the repository, as that's where we'll pull in the Chromium sources we need :smile:.

**Comment by ItIsOHM** [comment]
Ahh I see! Thanks for the detailed answer @fallaciousreasoning! As you said, yes the chromium source code is huge (65-70 GiB iirc) and that will take some good amount of time to download on my end, so I'll report back in a few days if that's alright? 😅

**Comment by El-47** [comment]
@fallaciousreasoning can you assign me any beginner friendly issue?

**Comment by ItIsOHM** [comment]
@El-47 if you want you can take up this issue, as I'm not able to clone chromium on my machine.

**Comment by El-47** [comment]
ok i will take the issue, thanks.

**Comment by KSSaiTeja** [comment]
@GeetaSarvadnya Please assign this to me, i can work upon it and i will promise i can figure it out in 1 day

**Comment by fallaciousreasoning** [comment]
@KSSaiTeja I think that's up to @El-47 sorry, as they've already started work on it.

**Comment by El-47** [comment]
@KSSaiTeja you can take the issue

**Comment by El-47** [comment]
@fallaciousreasoning there was a problem in cloning chromium on my pc

**Comment by KSSaiTeja** [comment]
> @KSSaiTeja you can take the issue

I'll start work on today

please tell me clearly how can i reproduce this issue

**Comment by GeetaSarvadnya** [comment]
@KSSaiTeja The steps to reproduce the issue are given in the https://github.com/brave/brave-browser/issues/31355#issue-1778145667

**Comment by KSSaiTeja** [comment]
@GeetaSarvadnya could you please tell me how can i switch to the `1.55.17` as i am beginner to opensource, i don't know abut this and also i faced a issue that there are no files in`src` folder.. could you please tell me clearly how to run the project locally and fix the issue

Your reply for this means a lot for me 😄

**Comment by GeetaSarvadnya** [comment]
@KSSaiTeja You candownload the specific brave version from Github via https://github.com/brave/brave-browser/releases the issue is reproducible even in the latest nightly/master build  `1.59.24` 

Reg the `src` folder empty I can't help much, looks like project cloning issue from your end. @fallaciousreasoning Could you assist @KSSaiTeja if possible? Thanks!

**Comment by fallaciousreasoning** [comment]
The instructions for cloning the repository are further up the thread, in this comment: https://github.com/brave/brave-browser/issues/31355#issuecomment-1664899280

You should fix the issue in the `master` branch and submit a PR from your fork.

**Comment by KSSaiTeja** [comment]
ok i’ll try to fork the master branch and install the above resources

**Comment by KSSaiTeja** [comment]
<img width="1091" alt="Screenshot 2023-08-18 at 10 31 18 AM" src="https://github.com/brave/brave-browser/assets/130981122/0fe4d798-5fb0-476f-8f15-6eca19572348">


Can you please help me out. i got this error   when running that command

**Comment by KSSaiTeja** [comment]
i think i am not able to work on this issue.. please unassign me and assign others to do this work. i can work upon another issue. please do unassign me..!

**Comment by rizzabh** [comment]
Hey I wanna give it a try.

**Comment by fallaciousreasoning** [comment]
Feel free!

**Comment by KulmeetSJ** [comment]
Hey can you please assign this issue to me

**Comment by KSSaiTeja** [comment]
Now i have developed prior skills to set the project. could you please assign this to me?

**Comment by kjozwiak** [comment]
The above requires `1.59.101` or higher for `1.59.x` verification 👍

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.59.102 Chromium: 117.0.5938.140 (Official Build) beta (arm64)
-- | --
Revision | 11284b6d5d51252fe1a6fb6b5cba35ef98982357
OS | macOS Version 14.0 (Build 23A344)
```
